core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -140,6 +140,8 @@
     public static final String ONCOKB_PUBLIC_API_URL = "oncokb.public_api.url";
     public static final String SHOW_ONCOKB = "show.oncokb";
 
+    public static final String SHOW_TWITTER_BOOKMARK = "bookmark.twitterLink.enabled";
```

Comment:
@Tamba32 Can you either
-  add the `@Value{$bookmark.twitter_link.enabled}` decorator? That way we can change the value on runtime with `-d bookmark.twitter_link.enabled=true` or `-d bookmark.twitter_link.enabled=false`. -- Or -
- another way might be to use  bookmark.twitter_link.enabled directly in the JSP w/o using GlobalProperties class (@ersinciftci might know).

Also for consistency you prolly don't want to use camelCase on the properties, they all use underscores and dots.